native/monogame/desktopvk.vcxproj

```diff
@@ -0,0 +1,131 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This was not intended to be committed as it is generated by premake.  @harry-cpp ?

Reply:
I agree, I do not think this file should have been added, please revert